modules/dxp/apps/portal-search-tuning/portal-search-tuning-synonyms-web/src/main/java/com/liferay/portal/search/tuning/synonyms/web/internal/portlet/action/EditSynonymSetsMVCActionCommand.java

```diff
@@ -88,19 +88,20 @@ protected void updateSynonymSet(ActionRequest actionRequest)
 
 	protected void updateSynonymSetIndex(
 			SynonymSetIndexName synonymSetIndexName, String synonyms,
-			Optional<SynonymSet> synonymSetOptional)
+			SynonymSet synonymSet)
 		throws PortalException {
 
 		SynonymSet.SynonymSetBuilder synonymSetBuilder =
 			new SynonymSet.SynonymSetBuilder();
 
 		synonymSetBuilder.synonyms(synonyms);
 
-		synonymSetOptional.ifPresent(
-			synonymSet -> synonymSetBuilder.synonymSetDocumentId(
-				synonymSet.getSynonymSetDocumentId()));
+		if (synonymSet != null) {
+			synonymSetBuilder.synonymSetDocumentId(
+				synonymSet.getSynonymSetDocumentId());
+		}
 
-		if (synonymSetOptional.isPresent()) {
+		if (synonymSet != null) {
```

Comment:
@LisztLiszt These 2 `if` are with same condition, please merge them.

Reply:
@LisztLiszt After you merge the 2 `if`, there will be another issue. Usually we prefer `if-else` over `if_not-else`. So please flip the condition.